.cirrus.yml

```diff
@@ -114,3 +115,10 @@ test_windows_task:
   test_erlang_script: make --keep-going test_erlang
 
   test_elixir_script: make --keep-going test_elixir
+
+  check_reproducible_script: |
+    if [ -n "$CHECK_REPRODUCIBLE" ]; then
+      make check_reproducible
+    else
+      echo "The format reproducibility of the build is only checked in the last stable Erlang/OTP version."
```

Comment:
```suggestion
      echo "The reproducibility of the build is only checked in the last stable Erlang/OTP version."
```

Reply:
thank you. I'm fixing it in master as well.